CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and as of version 1.0.0, follows semantic versioning.
 
 ## [Unreleased]
+  * Switched Call Ready TAR Median Target Coverage to Mean Bait Coverage
+  * No data for swap view doesn't cause crash (stage cache has become empty)
```

Comment:
These changes would optimally be in different commits so that the git blame for the relevant lines aren't confusing down the line